### DISCUSSION TITLE
Syndromic disease

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#2676] [Mondo design decisions]
**Mondo term (ID and Label)**
MONDO_0002254 'syndromic disease'

**Suggested revision of textual definition. Please include a source, like a PubMed ID (in the format PMID:#######)**
A collection of signs and symptoms associated with a specific health-related cause, where the underlying pathology/cause is not known.
source:
https://www.healthwriterhub.com/disease-disorder-condition-syndrome-whats-the-difference/
https://orcid.org/0000-0001-7090-9648

cc @PabloBotas, @cmungall, @julianig72, @zoependlington, @paolaroncaglia 
Do you all agree with this revised def?

**Comment by cmungall** [comment]
I don't agree or disagree, I think that agreeing on a definition of syndrome will be impossible

However, note that we can't just change the text definition unilaterally. There is a logical definition here and the change to the text definition becomes inconsistent. It also becomes inconsistent with the way things are placed there just now. We'd essentially need to go through the whole ontology and re-decide. We also need to make sure this is consistent with OMIM's terminology when they classify disorders and syndromic vs non-syndromic. Changes at the upper level can't be made in isolation!

One of the advantages of the current definition is that is based on objective largely static criteria (albeit with some fuzziness over what is a cluster of distinct symptoms vs a unary one). Changing this to a definition that incorporates epistemology will entail a need to constantly change the ontology as more is learned about etiology of disease, until all causes are known, and all diseases are moved out from under "syndrome". So I am not in favor of the change based on maintenance reasons.

I could get behind a proposal to obsolete syndrome. If people are interested in the etiology (e.g show me all diseases without known causes) they can just query the annotations. If people are interested in another interpretation of syndrome, they can just ask the right query (e.g. diseases with multiple distinct phenotypes). No need for bucket classes

**Comment by paolaroncaglia** [comment]
+1 from me to obsoletion of 'syndromic disease' (hard to define, hard to maintain, so many children I question its utility; terms labelled "syndrome with X" and "syndrome without X" are among the worst bearers of incorrect superclass/subclass axioms, I keep coming across examples).
Of course, this is probably a can of worms. :-) For one, you would then also have to look at all terms labelled "non-syndromic X".

Either way, for non-ontology savvy users interested in the use-cases above, maybe you could design appropriate queries and make them available in Monarch, if you don't offer them already? Similarly to some of the AmiGO Lab queries.

**Comment by nicolevasilevsky** [comment]
> Either way, for non-ontology savvy users interested in the use-cases above, maybe you could design appropriate queries and make them available in Monarch, if you don't offer them already?

I love this idea!

**Comment by nicolevasilevsky** [comment]
I am in favor of obsoleting too. It would be a bit of work to rehome the orphan classes but I am happy to work on that.
